### PR TITLE
feat: implement eth_blobBaseFee endpoint

### LINKED ISSUE #292: Implement `eth_blobBaseFee` RPC endpoint
https://github.com/ethereum/execution-apis/blob/main/src/eth/fee_market.yaml#L9-L16

### LINKED ISSUE #325: Hive Testing: `eth_blobBaseFee`
- [ ]   blob-gas-price

### PR DESCRIPTION
**Pull request by avilagaston9**
**Motivation**

We have implemented `calculate_base_fee_per_blob_gas`. We are able to implement this endpoint now.

**Description**

Implements `eth_blobBaseFee` endpoint.

Closes #292 
Closes #325

### CODE REVIEW

File: crates/rpc/eth/block.rs [hunk 1]
```diff
@@ -276,6 +276,25 @@ pub fn block_number(storage: Store) -> Result<Value, RpcErr> {
     }
 }
 
+pub fn get_blob_base_fee(storage: &Store) -> Result<Value, RpcErr> {
+    info!("Requested blob gas price");
+    match storage.get_latest_block_number() {
+        Ok(Some(block_number)) => {
+            let header = match storage.get_block_header(block_number)? {
```
Comment (mpaulucci) [inline crates/rpc/eth/block.rs:283]: shouldn't you be able to just do let `header = match storage.get_block_header(block_number)?`?

Comment (avilagaston9) [reply crates/rpc/eth/block.rs:270]: You mean that we should return just `Ok(block_header)` instead of `Ok(Some(block_header))` from Storage?

Comment (mpaulucci) [reply crates/rpc/eth/block.rs:270]: See https://github.com/lambdaclass/ethereum_rust/blob/main/crates/rpc/eth/block.rs#L93

Comment (avilagaston9) [reply crates/rpc/eth/block.rs:270]: The `match` in that case is two lines below that one 🤔

**Review by mpaulucci — APPROVED** [review]

**Pull request merged**